C/07_cProgramming.c

```diff
@@ -7,7 +7,8 @@ int main() {
 	int currentYear;
 	int birthYear;
 	
-	currentYear = 2014;
```

Comment:
`#include<time.h>` directives could be used for current year.
or can use predefined macros in C.
```c
#include <stdio.h>
int main()
{
    printf("%s", __DATE__ + 7); //points to 8th byte to print year
    return 0;
}
```